### PR TITLE
Add pytorch-dp

### PR DESCRIPTION
**Pull request by ndmaxar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ndmaxar** [comment]
@rluria14  and @oblute ok to be maintainers?

File: recipes/pytorch-dp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  skip: True  # [win or not py37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - torchvision
```
Comment (hmaarrfk) [inline recipes/pytorch-dp/meta.yaml:20]: there has been ALOT of talk of removing the pytorch stack on conda-forge.

We:
1. Don't have enough time to compile it all on CIs
2. Have yet to compile it for use with GPUs.

Maybe you could help us update the pytorch stack? or remove this dependency?

Comment (ndmaxar) [reply recipes/pytorch-dp/meta.yaml:20]: I'd be happy to help where I can. But I don't think we can remove torchvision as a dependency.

Comment (scopatz) [reply recipes/pytorch-dp/meta.yaml:20]: Is torchvision needed to build the package?

**Comment by ndmaxar** [comment]
@conda-forge/staged-recipes this recipe is ready for review.

File: recipes/pytorch-dp/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/pytorch-dp/meta.yaml:20]: It is unlikely torchvision is needed as to build

File: recipes/pytorch-dp/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+echo "numpy>=1.15
+torch>=1.3
+torchvision>=0.4
+tqdm>=4.40
+scipy>=1.2
+pytest" > requirements.txt
```
Comment (scopatz) [inline recipes/pytorch-dp/build.sh:6]: Woah.... this is weird and wrong.

File: recipes/pytorch-dp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/{{ name }}/archive/v{{ version_long }}.tar.gz
+  sha256: 77f01cb86183e55c402bdaa05aa52542418c5bea1cfa3f68fabc981c9e1680be
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```
Comment (scopatz) [inline recipes/pytorch-dp/meta.yaml:16]: This can be noarch: python.  But please remove this line then

Comment (ndmaxar) [reply recipes/pytorch-dp/meta.yaml]: I've added the noarch: python and removed the skip.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**